app-android-xamarin/Moodis/Moodis/Feature/CameraFeature/CameraFragment.cs

```diff
@@ -0,0 +1,101 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+using Android.App;
```

Comment:
use this `using Android.Support.V4.App;` instead of `Android.app`, so backwards compatibility with older Android versions and newest updates to Android libraries would be ensured

Reply:
line 17. [Activity(Label = "Camera activity")] uses Android.App;